webapp/src/DataFixtures/DefaultData/ExecutableFixture.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+
+namespace App\DataFixtures\DefaultData;
+
+use App\Entity\Executable;
+use App\Entity\ImmutableExecutable;
+use App\Service\DOMJudgeService;
+use Doctrine\Persistence\ObjectManager;
+use Psr\Log\LoggerInterface;
+use ZipArchive;
+
+class ExecutableFixture extends AbstractDefaultDataFixture
+{
+    /**
+     * @var string
+     */
+    protected $sqlDir;
+
+    /**
+     * @var DOMJudgeService
+     */
+    protected $dj;
+
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
+
+    public function __construct(string $sqlDir, DOMJudgeService $dj, LoggerInterface $logger)
+    {
+        $this->sqlDir = $sqlDir;
+        $this->dj     = $dj;
+        $this->logger = $logger;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function load(ObjectManager $manager)
+    {
+        $data = [
+            // ID,         description,               type
+            ['compare',    'default compare script', 'compare'],
+            ['java_javac', 'java_javac',             'compile'],
```

Comment:
Is it better to make compile_script in the Languages fixture an array?

Reply:
We could, but then which one do we link? The first one?
I'm not sure if that's better than having this one here.